docs/index.md

```diff
@@ -8,7 +8,11 @@ description: |-
 
 # Coder Provider
 
-Terraform provider for managing Coder [templates](https://coder.com/docs/templates), which are the underlying infrastructure for Coder [workspaces](https://coder.com/docs/workspaces).
+Terraform provider for managing Coder [templates](https://coder.com/docs/admin/templates), which are the underlying infrastructure for Coder [workspaces](https://coder.com/docs/user-guides/workspace-management).
+
+-> Requires Coder v2.13.0 or later.
```

Comment:
We introduced a few attributes in the v2.13.0 release.

Reply:
`coder_workspace_owner.login_type` is only available in v2.18.0 or later.